unit/testing-utils/require_goto_statements.cpp

```diff
@@ -33,6 +34,18 @@ require_goto_statements::get_all_statements(const exprt &function_value)
   return statements;
 }
 
+/// \param symbol_table Symbol table for the test
+/// \return All codet statements of the __CPROVER_start function
+const std::vector<codet> require_goto_statements::get_entry_point_statements(
+  const symbol_tablet &symbol_table)
+{
+  // Retrieve __CPROVER_start
+  const symbolt *entry_point_function =
+    symbol_table.lookup(goto_functionst::entry_point());
+
```

Comment:
To avoid crashing probably worth putting a `REQUIRE(entry_point_function != nullptr)` in here